build/wdqs-frontend/default.conf

```diff
@@ -23,3 +13,40 @@ server {
         root   /usr/share/nginx/html;
     }
 }
+
```

Comment:
What is the purpose of this optional commented-out section of server config? Does it need a comment or clarification here?

Reply:
Well, no, that is just a left over. Will be removed. Thanks for catching that.